package.json

```diff
@@ -64,6 +64,7 @@
     "grunt-stylelint": "^0.20.1",
     "grunt-terser": "^2.0.0",
     "grunt-webpack": "^7.0.0",
+    "jiti": "^2.4.2",
```

Comment:
Why is this new dependency needed?

Reply:
This is so we can use a ts config for eslint instead of js. So `eslint.config.ts` instead of `eslint.config.js`

Reply:
Looking at the jiti docs, they suggest it's built-in to eslint? Why do we need to depend on it explicitly?

I've been using `tsx` for this purpose in other projects, maybe I'll see if `jiti` is better 🤔

Reply:
According to the [docs](https://eslint.org/docs/latest/use/configure/configuration-files#typescript-configuration-files) it's better to install ziti